client-app/shared/layout/components/language-selector/language-selector.vue

```diff
@@ -11,9 +11,12 @@
       ></span>
 
       <span
-        class="fi fis fa-2x -my-3 h-[30px] !w-[30px] rounded-full lg:h-[14px] lg:!w-[14px]"
-        :class="`fi-${currentLanguage?.twoLetterRegionName.toLowerCase()}`"
-      ></span>
+        class="-my-3 flex h-[30px] !w-[30px] items-center justify-center overflow-hidden rounded-full lg:h-[14px] lg:!w-[14px]"
+      >
+        <span class="country-flag text-[40px] lg:text-[18px]">{{
```

Comment:
text-6xl lg:text-[1.75rem]

Reply:
they are a bit different numbers in your example, but I got the idea and changed values